### ISSUE TITLE
No Output results in Step3

### ISSUE THREAD

**Issue by aghasaadmohammad** [b01902041/Deep-Virtual-Try-on-with-Clothes-Transform#7]
Blank black images in results after transformation step(Step3)

**Comment by b0-dotcom** [comment]
do you have any idea how to get in touch with the person who did this I want to ask something

**Comment by aghasaadmohammad** [comment]
no i havent, i didnt got any reply

**Comment by b0-dotcom** [comment]
are you familiar with what he did, like can you do the same project he did   
  

  
  

  
  
>

**Comment by aghasaadmohammad** [comment]
Yeah i did follow his work , and tried to improve it but i couldn't get to a point but yes, i am willing to work over it

**Comment by b0-dotcom** [comment]
so basically I have had this idea before I saw his work however I never found the right people who know a good deal of python... here I am. So what I am doing is I am trying to open up a clothing line with this technology. now I am willing to pay you for helping me out on this project , do you have whatsapp or anywhere where I could introduce myself and If you could tell me where are you from?   
  

  
  

  
  
>

**Comment by b0-dotcom** [comment]
can you message me on email

**Comment by b0-dotcom** [comment]
text me on my email....   
  

  
  

  
  
>

**Comment by thotd21** [comment]
> Blank black images in results after transformation step(Step3)

Hi @aghasaadmohammad,
Have you finished fixing this error yet? I am having a similar situation.
And can you send me an example of the data train you are using: 1 pair of images imgs_train, imgs_train_mask, and imgs_train_label.
Thanks and best regards.

**Comment by kuldeepbishnoi** [comment]
Any idea how where to download .h5 file?
`
OSError: Unable to open file (unable to open file: name = './models_512/netG1527659649.3044329.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
`
